cmd/rad/cmd/root.go

```diff
@@ -482,3 +512,22 @@ func getRootSpanName() string {
 		return args[0]
 	}
 }
+
+// wirePreviewSubcommand adds a --preview flag and routes RunE to the preview command when set.
+func wirePreviewSubcommand(cmd *cobra.Command, previewCmd *cobra.Command) {
+	cmd.Flags().Bool("preview", false, "Use the Radius.Core preview implementation")
```

Comment:
Is this applicable to any resource type we want to support preview for in the future?